src/Avalonia.Controls/Presenters/ContentPresenter.cs

```diff
@@ -159,6 +226,105 @@ public BoxShadows BoxShadow
             set => SetValue(BoxShadowProperty, value);
         }
 
+        /// <summary>
+        /// Gets or sets a brush used to paint the text.
+        /// </summary>
```

Comment:
Since all of these properties come from TextElement or TextBlock it's a good idea to inherit the comments as well using `<inheritdoc cref="TextElement.Foreground" />` This avoids duplicated code and ensures the docs/comments are always in sync.

Reply:
@robloo TIL :)